src/sql/workbench/parts/query/test/browser/queryActions.test.ts

```diff
@@ -317,38 +317,6 @@ suite('SQL QueryAction Tests', () => {
 		assert.equal(runQuerySelection.endColumn, selectionToReturnInGetSelection.endColumn, 'endColumn should match');
 	});
 
-	test('CancelQueryAction calls cancelQuery() only if URI is connected', (done) => {
-		// ... Create assert variables
```

Comment:
Shouldn't this be converted over to see if runner.cancelQuery is called?

Reply:
I guess I could, but I would have had to mock a input, that then mocks a runner, but theres not much point, since the query runner is designed to handle cancels correctly. This can cancel the query incorrectly all it wants and there is no negative impact.